packages/zowe-explorer/__tests__/__e2e__/step_definitions/dialogs/IssueCommand.steps.ts

```diff
@@ -20,22 +20,12 @@ Then("a quick pick appears to select a profile", async function () {
 });
 When("a user selects a profile", async function () {
     await expect(this.input).toBeDefined();
-    let qpItems = await this.input.getQuickPicks();
+    const qpItems = await this.input.getQuickPicks();
     await qpItems.at(0).select();
     await expect(this.input.elem).toBeDisplayedInViewport();
-
-    // Issue TSO command has an extra quick pick to select from
-    if (this.openedCommand === "Issue TSO Command") {
-        qpItems = await this.input.getQuickPicks();
-        await qpItems.at(0).select();
-    }
 });
 
 Then(/a user can enter in (.*) as the command and submit it/, async function (command: string) {
-    if (this.openedCommand === "Issue TSO Command") {
-        await this.input.setText("IZUACCT");
-        await this.input.confirm();
-    }
```

Comment:
I assume we were able to remove this because the TSO account is expected to be defined in zowe.config.json. Should we mention something about setting up a team config in the readme for e2e tests?

Reply:
Yep, that's correct - I've added a section to the README for e2e tests to clarify how to build a valid config for testing.